File: HW/HW11/hw11Test.py
import unittest
from hw11 import *

class MyTestCase(unittest.TestCase):
    def test_something(self):
        img = [[1, 1, 1],
               [1, 1, 0],
               [1, 0, 1]]
        hw11Util.floodFill(self, img, 1, 1, 2)

        for i in range(len(img)):
            for j in range(len(img[0])):
                print(str(img[i][j]), end = " ")
            print()

        img = [[0, 0, 0],
               [0, 0, 0],
               [0, 0, 0]]
        hw11Util.floodFill(self, img, 0, 0, 0)
        print()

        for i in range(len(img)):
            for j in range(len(img[0])):
                print(str(img[i][j]), end=" ")
            print()


if __name__ == '__main__':
    unittest.main()

File: HW/HW11/hw11Test.java
import org.junit.Test;

import static org.junit.Assert.*;

public class hw11Test {
    @Test
    public void test() {
        int[][] img = {
                {1, 1, 1},
                {1, 1, 0},
                {1, 0, 1}
        };

        hw11.floodfill(img, 1, 1, 2);
        for (int i = 0; i < img.length; i++) {
            for (int j = 0; j < img[0].length; j++) {
                System.out.print(img[i][j] + " ");
            }
            System.out.println();
        }

        int[][] img2 = {
                {0, 0, 0},
                {0, 0, 0},
                {0, 0, 0}
        };

        hw11.floodfill(img2, 0, 0, 0);
        System.out.println();

        for (int i = 0; i < img2.length; i++) {
            for (int j = 0; j < img2[0].length; j++) {
                System.out.print(img2[i][j] + " ");
            }
            System.out.println();
        }

    }

}